### ISSUE TITLE
What's worth analyzing that's relevant to the pandemic?

### ISSUE THREAD

**Issue by umran** [umran/pandemic#1]
let's use this issue as a space to list down potential topics to analyze.

**Comment by doublebigmak** [comment]
We can either look at this from a top down or bottom up perspective.

Top down -> look at measures of GDP, drivers of GDP, forecast as you would normally, without exogenous shocks. Then add in sensitivity effects to different sectors - note that if you're looking at Canada, would need to research effects on fx, oil, etc. during this period. 

Bottom up -> agent modelling, simulating a supply and demand market at a "steady state" and try to manipulate situation with effects. Take those observations and apply to a bigger picture. Much harder to do although agent simulation is interesting.

**Comment by umran** [comment]
Would be nice to have some top down analysis in the pipeline while we take a crack at agent modelling.

Re top down approach: to take the GDP example do you mean looking at the the primary components C + I + G + NX as endogenous variables?

**Comment by doublebigmak** [comment]
yes that would be a way to measure GDP, I'd also look at factors like labour, employment, etc, measures of disposable income and how it relates. I think those are the main drivers of the economy, at least in conjunction with capital
